src/components/ebay-snackbar-dialog/examples/02-action/template.marko

```diff
@@ -0,0 +1,23 @@
+<ebay-snackbar-dialog open=state.open on-close('closeSnackbar') on-action("closeSnackbar")>
+    This is the action snackbar
+    <@action access-key="U">
+        Undo
+    </@action>
+</ebay-snackbar-dialog>
+<ebay-button on-click("openSnackbar")>
```

Comment:
In all examples, can you put the button that launches the snackbar before the dialog. This will ensure correct reading & keyboard order.

Reply:
done